### ISSUE TITLE
Since version 1.3.2 action started failing on macOS GitHub runners.

### ISSUE THREAD

**Issue by mvolpato** [kielabokkie/ssh-key-and-known-hosts-action#24]
On post job cleanup I get this error:

```
Post job cleanup.
Stopping ssh-agent
/Users/runner/work/_actions/kielabokkie/ssh-key-and-known-hosts-action/v1.3.4/dist/cleanup/index.js:1368
		error = new Error(message);
		        ^

Error: Command failed with exit code 1: kill -9 $(pidof ssh-agent)
/bin/sh: pidof: command not found
kill: usage: kill [-s sigspec | -n signum | -sigspec] pid | jobspec ... or kill -l [sigspec]
    at makeError (/Users/runner/work/_actions/kielabokkie/ssh-key-and-known-hosts-action/v1.3.4/dist/cleanup/index.js:1368:11)
    at handlePromise (/Users/runner/work/_actions/kielabokkie/ssh-key-and-known-hosts-action/v1.3.4/dist/cleanup/index.js:[2](REDACTED#step:47:2)722:26)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async run (/Users/runner/work/_actions/kielabokkie/ssh-key-and-known-hosts-action/v1.[3](REDACTED#step:47:3).[4](REDACTED#step:47:4)/dist/cleanup/index.js:64[5](REDACTED#step:47:5):3) {
  shortMessage: 'Command failed with exit code 1: kill -[9](REDACTED#step:47:9) $(pidof ssh-agent)',
  command: 'kill -9 $(pidof ssh-agent)',
  exitCode: 1,
  signal: undefined,
  signalDescription: undefined,
  stdout: '',
  stderr: '/bin/sh: pidof: command not found\n' +
    'kill: usage: kill [-s sigspec | -n signum | -sigspec] pid | jobspec ... or kill -l [sigspec]',
  failed: true,
  timedOut: false,
  isCanceled: false,
  killed: false
}
```

Before version 1.3.2, `pidof: command not found` was still an issue, but the entire workflow was not failing.

**Comment by kielabokkie** [comment]
Thanks for reporting @mvolpato. I'll have to see how I can reliably get the Process ID on different OS'es. Version `1.3.2` was an update to make sure the action runs on Node 16 instead of 12 so it must be more strict when it comes to errors.

**Comment by mvolpato** [comment]
Would this help?
`ps -e | grep -m1 ssh-agent | awk '{print $1}'`

**Comment by kielabokkie** [comment]
@mvolpato I've just released [v1.3.5](https://github.com/kielabokkie/ssh-key-and-known-hosts-action/releases/tag/v1.3.5). Would appreciate it if you can give it a go and let me know if it works now.

**Comment by mvolpato** [comment]
> @mvolpato I've just released [v1.3.5](https://github.com/kielabokkie/ssh-key-and-known-hosts-action/releases/tag/v1.3.5). Would appreciate it if you can give it a go and let me know if it works now.

Thanks @kielabokkie, I tested it with one of my workflows and it works!

**Comment by shango420** [comment]
>
